coderd/portsharing/portsharing.go

```diff
@@ -10,6 +10,7 @@ import (
 type PortSharer interface {
```

Comment:
We should remove PortSharing from all of these interface function names.

Reply:
Sure thing